packages/toolpad-components/src/Text.tsx

```diff
@@ -23,9 +22,18 @@ interface TextProps extends Omit<BaseProps, 'children'> {
 
 const MarkdownContainer = styled('div')({
   display: 'block',
-  [`&:empty::before`]: { content: '""', display: 'inline-block' },
+  minHeight: '25px',
 });
 
+const CodeContainer = styled('pre')(({ theme }) => ({
+  whiteSpace: 'pre-wrap',
```

Comment:
Most code doesn't look great when wrapped. Can't we handle this using overflow?

Reply:
`white-space: pre-wrap` 
> Sequences of white space are preserved. Lines are broken at newline characters, at `<br>`, and as necessary to fill line boxes.

solves the problem acceptably well in most code snippets I tried.